src/popover/types.js

```diff
@@ -80,6 +80,10 @@ export type BasePopoverPropsT = {
   mountNode?: HTMLElement,
   /** How long should be fade out animation in ms, default 0ms */
   animateOutTime?: number,
+  /** Popper options override
+   * https://popper.js.org/popper-documentation.html#Popper.Defaults */
+  // eslint-disable-next-line flowtype/no-weak-types
+  popperOptions?: any,
```

Comment:
popper js provides flow types [here](https://github.com/FezVrasta/popper.js/blob/master/packages/popper/index.js.flow#L104) can that be used?

Reply:
How does this work with `flow`. Can I do something like `import typeof {Options} from 'popper.js'`? Or will I need to copy the entire module over into the `Layer` types file?

Reply:
the syntax is probably `import type {Options} from 'popper.js';`

Reply:
I wasn't 100% sure it was working. There is one file with the flow decs under `/node_modules/popper.js/dist/esm/poppper.js.flow`(intentional 3 ps) that has a module declaration of `Popper`. `Options` is within that module. I thought it might be a straight import but my IntelliSense didn't pick up on anything.